### PR TITLE
📚 Source Shopify: migrate to new sandbox, update API version to 2021-07

### LINKED ISSUE #4631: Use new shopify sandbox in CI
## Tell us about the problem you're trying to solve
We are currently using a real Shopify store for CI testing. We should instead use the one prepared in https://github.com/airbytehq/airbyte/issues/4496 in

Acceptance Criteria: 
1. Verify tests pass when the new credentials are used
2. Update creds in CI

### PR DESCRIPTION
**Pull request by bazarnov**
## What
#4631 - Use new shopify sandbox in CI

## How
1) Migrated to new sandbox account `airbyte-integration-test`
2) Edited the next stream schemas: `abandoned_checkouts, orders, price_rules, transactions` due to major API version change from `2021-04` to `2021-07` (some data-types now return `string` instead of `number`)
3) Allow connector to use new API version `2021-07`
4) Updated `acceptance-tests`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [X] Issue acceptance criteria met
- [X] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [x] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [X] Unit & integration tests added as appropriate (and are passing)
      `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
- [x] Code reviews completed
- [x] Documentation updated 
    - [X] Created or updated reference docs in `docs/integrations/<source or destination>/<name>`.
    - [X] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [X] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [X] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [x] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [X] No major blockers
- [ ] PR merged into master branch
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-shopify/integration_tests/abnormal_state.json [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-    "updated_at": "2020-08-03T14:42:06-07:00"
+    "updated_at": "2021-07-19T06:52:06-07:00"
   },
   "draft_orders": {
-    "updated_at": "2020-06-08T21:09:25-07:00"
+    "updated_at": "2021-07-07T08:18:59-07:00"
   },
   "products": {
-    "updated_at": "2021-06-25T06:13:37-07:00"
+    "updated_at": "2021-07-19T06:56:06-07:00"
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-shopify/integration_tests/abnormal_state.json:12]: should we put this very very far into the future? 2024? feels like we update this every time we make a shopify change

File: docs/integrations/sources/shopify.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ Shopify has some [rate limit restrictions](https://shopify.dev/concepts/about-ap
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.10  | 2021-07-19 | [4830](https://github.com/airbytehq/airbyte/pull/4830) | Fix for streams json schemas, allow to use API version 2021-07 |
```
Comment (sherifnada) [inline docs/integrations/sources/shopify.md:64]:
```suggestion
| 0.1.10  | 2021-07-19 | [4830](https://github.com/airbytehq/airbyte/pull/4830) | Fix for streams json schemas, upgrade to API version 2021-07 |
```

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-shopify/integration_tests/abnormal_state.json (hunk 1 above)
Comment (bazarnov) [reply airbyte-integrations/connectors/source-shopify/integration_tests/abnormal_state.json:12]: Updated.

File: docs/integrations/sources/shopify.md (hunk 2 above)
Comment (bazarnov) [reply docs/integrations/sources/shopify.md:64]: Commited.

**Review by TymoshokDmytro — APPROVED** [review]

**Pull request merged**